Dockerfile.production

```diff
@@ -10,7 +10,7 @@ WORKDIR /osem/
 RUN export NOKOGIRI_USE_SYSTEM_LIBRARIES=1; bundle install --jobs=3 --retry=3 --without test development
 
 # Generate assets
-RUN export RAILS_ENV=production; bundle exec rake assets:clobber assets:precompile
+RUN export RAILS_ENV=production; bundle exec rake assets:precompile
```

Comment:
If you don't remove them first, you might run into some dangling assets.

Reply:
Without `public/assets` and `tmp` copied into the image there shouldn’t be anything to remove.

Reply:
true! sorry